platform/admin/core/admin-core-alertservice/src/test/groovy/org/codice/ddf/admin/core/alert/service/internal/AlertServiceBeanTest.groovy

```diff
@@ -0,0 +1,264 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.core.alert.service.internal
+
+import org.codice.ddf.admin.core.alert.service.data.internal.Alert
+import org.osgi.service.event.Event
+import spock.lang.Specification
+
+import javax.management.MalformedObjectNameException
+import javax.management.ObjectName
+import java.lang.management.ManagementFactory
+
+class AlertServiceBeanTest extends Specification {
+
+    final mbeanName = new ObjectName(AlertServiceBean.class.getName() + ":service=alert-service")
+
+    def alertServiceBean
+
+    void setup() throws Exception {
+        // the keystore path must be set for the InsecureDefaultsService to use
+        System.setProperty("javax.net.ssl.keyStore", "TestKeystorePath")
+        alertServiceBean = new AlertServiceBean(ManagementFactory.getPlatformMBeanServer())
+        assert !isRegistered()
```

Comment:
❗️ It's usually better not to have any assertions in the setup method and instead have a separate test method to cover those assertions. Makes it easier to figure out what's being tested and what's broken when an assertion fails.